core/src/main/scala/kafka/server/DynamicBrokerConfig.scala

```diff
@@ -689,6 +690,20 @@ class DynamicAllowPreferredControllerFallback(server: KafkaServer) extends Broke
     }
   }
 }
+class LiCombinedControlRequestEnable(server: KafkaServer) extends BrokerReconfigurable {
+
+  override def reconfigurableConfigs: Set[String] = {
+    Set(KafkaConfig.LiCombinedControlRequestEnableProp)
+  }
+
+  override def validateReconfiguration(newConfig: KafkaConfig): Unit = {
+    // no additional validation is needed.
+  }
+
+  override def reconfigure(oldConfig: KafkaConfig, newConfig: KafkaConfig): Unit = {
+    server.kafkaController.controllerContext.setLiCombinedControlRequestEnabled(newConfig.liCombinedControlRequestEnable)
```

Comment:
is it safe to set this flag from the configuration thread instead of going through the controller's event queue to have the controller pick up the flag in its own event processing thread?

Reply:
upon closer look, I found that all I need is a boolean flag, and there is no need for further processing. That means the built-in dynamic config mechanism is already good enough, and there is no need to add an extra state in controller context.

Please take a look at the latest change. Thanks!